File: Programing_Basics_with_JS/exam28And29May2022/02.FootballKit.js
function footballKit(input) {
    let tshirtPrice = Number(input[0]);
    let moneyTillToWinBall = Number(input[1]);

    let pantsPrice = tshirtPrice * 0.75;
    let socksPrice = pantsPrice * 0.2;
    let shoosesPrice = (pantsPrice + tshirtPrice) * 2;
    let cost = tshirtPrice + pantsPrice + socksPrice + shoosesPrice;
    let totalCost = cost * 0.85;

    let neededMoney = moneyTillToWinBall - totalCost;

    if(totalCost >= moneyTillToWinBall) {
        console.log('Yes, he will earn the world-cup replica ball!');
        console.log(`His sum is ${totalCost.toFixed(2)} lv.`);
    } else {
        console.log('No, he will not earn the world-cup replica ball.');
        console.log(`He needs ${neededMoney.toFixed(2)} lv. more.`);
    }
}

footballKit(["25",
"100"])

footballKit(["55",
"310"])

footballKit(["59.99",
"500"])

File: Programming_Basics_with_Python/programming_basics_online_pre_exam_22_and_23_october_2022/02._football_kit.py
t_shirt_price = float(input())
win_price = float(input())

shorts_price = t_shirt_price * 0.75
socks_price = shorts_price * 0.2
soccer_cleats_price = (t_shirt_price + shorts_price) * 2

equipment_price = t_shirt_price + shorts_price + socks_price + soccer_cleats_price
total_equipment_price = equipment_price * 0.85

if total_equipment_price >= win_price:
    print('Yes, he will earn the world-cup replica ball!')
    print(f'His sum is {total_equipment_price:.2f} lv.')
else:
    print('No, he will not earn the world-cup replica ball.')
    print(f'He needs {win_price - total_equipment_price:.2f} lv. more.')